in_toto/verifylib.py

```diff
@@ -593,4 +593,84 @@ def verify_all_item_rules(items, links, target_links=None):
         link.products, target_links)
 
 
+def in_toto_verify(layout_path, layout_key_paths):
+  """
+  <Purpose>
+    Does entire in-toto supply chain verification of a final product
+    by performing the following actions:
+        1.  Load root layout as specified by 'layout_path'
+        2.  Load root layout signing key(s) (project owner public key(s)) as
+            specified by 'layout_key_paths'
```

Comment:
`layout_key_paths` is a list that contains paths to public keys whose private portions were used to sign the layout. A layout can be signed by multiple keys, right?